site/docs/guides/operations/kubernetes/healthchecks.md

```diff
@@ -0,0 +1,257 @@
+- tcpSocket -- this just checks for an open port.
+- grpc -- issues grpc call (we do no use grpc in egeria)
+- httpGet -- a simple GET. If return is >=200 and <400 it is successful
+- exec -- issues a specified command within the container
+- 
+## Example Egeria API calls
+
+In these examples the [httpie](https://httpie.io) tool will be used as it will print both the response code, and pretty-formatted body by default. Other tools like curl may also be used, but more parsing may be required of the responses.
+
+The examples here were run against the lab charts, using user 'garygeeke'. A simple security plugin is active which restricts user access to api calls.
+
+### Platform
+This checks if the *platform* is available.
+
+#### Platform is not running
+➜  ~ curl -k -X GET --connect-timeout 5 --max-time 5 "https://44623abc-eu-gb.lb.appdomain.cloud:9443/open-metadata/admin-services/users/admin/server-origin"
+Egeria OMAG Server Platform (version 4.1-SNAPSHOT)
+
+#### Platform is running
+➜  ~ http --verify=no --pretty=format GET "https://44623abc-eu-gb.lb.appdomain.cloud:9443/open-metadata/admin-services/users/admin/server-origin"
+HTTP/1.1 200
+Connection: keep-alive
+Content-Length: 42
+Content-Type: text/plain;charset=UTF-8
+Date: Thu, 18 May 2023 17:10:27 GMT
+Keep-Alive: timeout=60
+
+Egeria OMAG Server Platform (version 4.0)
+
+
+
+### Server
+This API call was introduced in [release 3.4](/release-notes/3-4.md)
+
+#### Server is not known
+```
+➜  ~ http --verify=no --pretty=format GET "https://44623abc-eu-gb.lb.appdomain.cloud:9443/open-metadata/admin-services/users/admin/servers/cocoMDS99/instance/status"
+HTTP/1.1 200
+Connection: keep-alive
+Content-Type: application/json
+Date: Thu, 18 May 2023 17:08:15 GMT
+Keep-Alive: timeout=60
+Transfer-Encoding: chunked
+
+{
+    "actionDescription": "getActiveServerStatus",
+    "class": "OMAGServerStatusResponse",
+    "exceptionClassName": "org.odpi.openmetadata.frameworks.connectors.ffdc.InvalidParameterException",
+    "exceptionErrorMessage": "OMAG-MULTI-TENANT-404-001 The OMAG Server cocoMDS99 is not available to service a request from user admin",
+    "exceptionErrorMessageId": "OMAG-MULTI-TENANT-404-001",
+    "exceptionErrorMessageParameters": [
+        "cocoMDS99",
+        "admin"
+    ],
+    "exceptionProperties": {
+        "parameterName": "serverName",
+        "serverName": "cocoMDS99"
+    },
+    "exceptionSystemAction": "The system is unable to process the request because the server is not running on the called platform.",
+    "exceptionUserAction": "Verify that the correct server is being called on the correct platform and that this server is running. Retry the request when the server is available.",
+    "relatedHTTPCode": 404
+}
+
+```
+#### No permission for api call
+```
+➜  ~ http --verify=no --pretty=format GET "https://44623abc-eu-gb.lb.appdomain.cloud:9443/open-metadata/admin-services/users/admin/servers/cocoMDS5/instance/status"
```

Comment:
This call is deprecated - should use the platform services request ` {baseURL}/open-metadata/platform-services/users/{user}/server-platform/servers/{serverName}/instance/status`